### PR TITLE
Security Considerations - Certification Path Validation

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

**Review by JAG-UK — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -727,7 +727,7 @@ An example of Signed Statement's payloads that can improve authenticity assuranc
...
 
 Auditors should be aware that the certification path information included in an unprotected `x5chain` header of a to-be-registered Signed Statement can be tampered with by a malicious Transparency Service (e.g., one that does not incorporate remote attestation), which may replace the intermediate certificates and ultimately connect to an unexpected root.
-This modification can allow malicious TS to forge Claims that look genuine except for the wrong trust anchor.
+This modification does, however, not allow an attacker to successfully launch a person-in-the-middle attack. A denial-of-service attack is, however, possible.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:730]:
```suggestion
This modification does, however, not allow an attacker to successfully launch a person-in-the-middle attack.
A denial-of-service attack is, however, possible. 
```

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by roywill — CHANGES_REQUESTED** [review]
The replacement loses the security issue and adds a new one.  

The modification of removing the certificate chain, which will save space in the ledger, opens up a problem of building the chain dynamically may result in the wrong trust anchor.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -727,7 +727,8 @@ An example of Signed Statement's payloads that can improve authenticity assuranc
...
 
 Auditors should be aware that the certification path information included in an unprotected `x5chain` header of a to-be-registered Signed Statement can be tampered with by a malicious Transparency Service (e.g., one that does not incorporate remote attestation), which may replace the intermediate certificates and ultimately connect to an unexpected root.
-This modification can allow malicious TS to forge Claims that look genuine except for the wrong trust anchor.
+This modification does, however, not allow an attacker to successfully launch a person-in-the-middle attack.
+A denial-of-service attack is, however, possible.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:731]:
```suggestion
This modification helps protect against person-in-the-middle attacks, but not denial-of-service.
```

Comment (OR13) [reply draft-ietf-scitt-architecture.md:731]: `,however` reads awk

**Comment by hannestschofenig** [comment]
Here is the status:

I do not believe there is a security problem. Roy says that Antoine thinks there is a problem. Antoine is, however, not at the meeting.

**Comment by henkbirkholz** [comment]
Antoine can object in the next iteration or during WGLC

**Pull request merged**